bpf/lib/conntrack_test.h

```diff
@@ -90,41 +101,60 @@ static void test___ct_lookup()
 			  &ct_state, true, seen_flags, &monitor);
 	assert(res == CT_ESTABLISHED);
 	assert(monitor == TRACE_PAYLOAD_LEN);
+	assert(timeout_in(entry, CT_SYN_TIMEOUT));
 
 	/* Second with no new flags, no advance in time is not monitored */
+	advance_time();
```

Comment:
nit: The comment immediately above this line says 'no advance in time', is now inconsistent with this line.

Reply:
Done. PTAL.